packages/seacas/applications/conjoin/CJ_ExodusFile.h

```diff
@@ -43,6 +43,8 @@ namespace Excn {
   public:
     explicit ExodusFile(size_t which);
     ~ExodusFile();
+    ExodusFile(const ExodusFile &) = delete;
+    ExodusFile operator=(const ExodusFile &) = delete;
```

Comment:
The usual idiom is to disable move construction and assignment as well.

Reply:
Yes, I should add those everywhere...